ruma-client-api/src/r0/account/add_3pid.rs

```diff
@@ -17,13 +17,13 @@ ruma_api! {
     request: {
         /// Additional information for the User-Interactive Authentication API.
         #[serde(skip_serializing_if = "Option::is_none")]
-        pub auth: Option<AuthData>,
```

Comment:
This should be `AuthData<'a>` at a later point (until then this should stay as `AuthData` rather than `&' a AuthData`).